mods/d2k/rules/aircraft.yaml

```
@@ -103,6 +109,10 @@ ornithopter:
 		HP: 9000
 	Armor:
 		Type: light
+	Encyclopedia:
+		Description: The fastest aircraft on Dune, the Ornithopther is lightly armored and capable of dropping 500lb bombs. This unit is most effective against infantry and lightly armored targets, but also damages armored targets.\n\nThey fly fast and are not targetable by anti-aircraft weapon.
```

```suggestion
		Description: The fastest aircraft on Dune, the Ornithopther is lightly armored and capable of dropping 500lb bombs. This unit is most effective against infantry and lightly armored targets, but also damages armored targets.
```
Ornithopters are targetable both with Missile tanks and Missile turrets